torch_xla/csrc/token_handler.cpp

```diff
@@ -30,8 +31,19 @@ xla::XlaOp SliceOneToken(xla::XlaOp input) {
 
 }  // namespace
 
+#ifndef PT_XLA_DISABLE_NUMERIC_CC_TOKEN_DEFAULT_VALUE
+#define PT_XLA_DISABLE_NUMERIC_CC_TOKEN_DEFAULT_VALUE false
+#endif
+
 xla::XlaOp TokenHandler::GetInput(xla::XlaOp input,
                                   const xla::Shape* input_shape) {
+  static bool disable_numeric_token =
+      xla::sys_util::GetEnvBool("PT_XLA_DISABLE_NUMERIC_CC_TOKEN",
```

Comment:
we usually don't prefix our env var with `PT_XLA`, maybe `DISABLE_NUMERIC_CC_TOKEN_DEFAULT_VALUE` directly?

Reply:
Done